### PR TITLE
chore: remove logrus timestamp format

### PR DESCRIPTION
**Pull request by luigibarbato**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -49,16 +49,26 @@ func exec() int {
...
+			DisableTimestamp: true,
+		},
+	}
+
+	log.SetLevel(logrus.InfoLevel)
+
 	h, err := os.Hostname()
 	if err != nil {
-		logrus.Debug(err)
+		log.Debug(err)
```
Comment (Al-Pragliola) [inline main.go:64]: after setting `log.SetLevel(logrus.InfoLevel)` we'll never see this debug message

File: main.go [hunk 2]
```diff
@@ -49,16 +49,26 @@ func exec() int {
...
 
 		h = "unknown"
 	}
 
 	t := os.Getenv("FURYCTL_MIXPANEL_TOKEN")
 	if t == "" {
-		logrus.Debug("FURYCTL_MIXPANEL_TOKEN is not set")
+		log.Debug("FURYCTL_MIXPANEL_TOKEN is not set")
```
Comment (Al-Pragliola) [inline main.go:71]: after setting `log.SetLevel(logrus.InfoLevel)` we'll never see this debug message

**Review by Al-Pragliola — COMMENTED** [review]
@ralgozino @luigibarbato

File: main.go (hunk 1 above)
Comment (luigibarbato) [reply main.go:64]: Should we rethink how we handle logrus logs in the cmd?

File: main.go (hunk 2 above)
Comment (luigibarbato) [reply main.go:71]: Should we rethink how we handle logrus logs in the cmd?

Comment (Al-Pragliola) [reply main.go:71]: mm maybe the whole logic is wrong, why are we doing that in the main.go?
```
	// Create the analytics tracker.
	a := analytics.NewTracker(t, versions[version], osArch, runtime.GOOS, "SIGHUP", h)
```

when we have in root.go the flag disable-analytics, shouldn't we wait for the flag to be parsed before creating a tracker?

**Comment by luigibarbato** [comment]
Force the Debug level as a workaround for now but we'll have to rethink how we handle logging in to the codebase.

**Review by Al-Pragliola — APPROVED** [review]
LGTM

**Pull request merged**